### ISSUE TITLE
SoX File Error

### ISSUE THREAD

**Issue by chonga2520** [Maxix25/keyboard-simulator#1]
I use Linux Mint 20.3. I ran `pip3 install pynput`, `pip3 install playsound`, and `apt install sox`. then ran `python3 main.py` and got this output:

Ready!
aWe found a error with SoX, did you install it?
b"play WARN alsa: can't encode 0-bit Unknown or not applicable\nplay FAIL formats: can't open input file `/sp_green/alpha/down/d6.mp3': No such file or directory\n"
We found a error with SoX, did you install it?
b"play WARN alsa: can't encode 0-bit Unknown or not applicable\nplay FAIL formats: can't open input file `/sp_green/alpha/up/u6.mp3': No such file or directory\n"
oWe found a error with SoX, did you install it?
b"play WARN alsa: can't encode 0-bit Unknown or not applicable\nplay FAIL formats: can't open input file `/sp_green/alpha/down/d7.mp3': No such file or directory\n"
We found a error with SoX, did you install it?
b"play WARN alsa: can't encode 0-bit Unknown or not applicable\nplay FAIL formats: can't open input file `/sp_green/alpha/up/u6.mp3': No such file or directory\n"

It seems to be some error with sox not being able to find the right filepath to the soundfiles. developer pls fix :(

Edit: added the pynput install command

**Comment by chonga2520** [comment]
Made a temporary fix. I'm glad I learned a bit of Python. I first fixed the filepath by changing the sound_path variable in main.py to `sound_path = "/home/user/keyboard-simulator-0.5" + __file__[:-7] + f"/{sound_name}/"` Then I ran `python3 main.py` again and it showed the following:

Ready!
aWe found a error with SoX, did you install it?
b"play WARN alsa: can't encode 0-bit Unknown or not applicable\nplay FAIL formats: no handler for file extension `mp3'\n"
We found a error with SoX, did you install it?
b"play WARN alsa: can't encode 0-bit Unknown or not applicable\nplay FAIL formats: no handler for file extension `mp3'\n"
We found a error with SoX, did you install it?
b"play WARN alsa: can't encode 0-bit Unknown or not applicable\nplay FAIL formats: no handler for file extension `mp3'\n"
We found a error with SoX, did you install it?
b"play WARN alsa: can't encode 0-bit Unknown or not applicable\nplay FAIL formats: no handler for file extension `mp3'\n"

After playing around with the sound_path variable again, I found this: [https://github.com/maximal/audio-waveform-php/issues/3](url). So i did what <a href="https://github.com/maximal">Maximal</a> said and ran `apt install libsox-fmt-all` and `apt install php-gd`. Then ran `python3 main.py` again and **voila, sound!** Now it shows only the following:

Ready!

This is only a temporary fix though, and should be fixed in the original code.
